doc/third-party/fivetran/setup-guide.md

```diff
@@ -0,0 +1,54 @@
+---
+name: Setup Guide
+title: Materialize Setup Guide
+description: Follow the guide to set up Materialize as a destination.
+---
+
+# Materialize Setup Guide {% badge text="Partner-Built" /%} {% badge text="Private Preview" /%}
+
+Follow the setup guide to connect Materialize to Fivetran.
+
+> NOTE: This destination is [partner-built](/docs/partner-built-program). For any questions related to Materialize destination and its documentation, contact [Materialize Support](mailto:support@materialize.com).
+
+---
+
+## Prerequisites
+
+To connect Materialize to Fivetran, you need the following:
+
+- A Materialize account with `CREATE` privileges on the database you're connecting to.
+- A Fivetran account with [permission to add destinations](/docs/using-fivetran/fivetran-dashboard/account-management/role-based-access-control#legacyandnewrbacmodel).
+
+---
+
+## Setup instructions
+
+### <span class="step-item"> Get Materialize credentials </span> 
+
+You must have a Materialize account with `CREATE` privileges on the database you're connecting to.
+
+1. Login to the [Materialize Console](https://console.materialize.com) with your account.
+2. Click "Connect externally" at the bottom of the sidebar.
+3. On the "External tools" tab, click "Create app password". 
+4. Copy the generated **password**, **Host**, and **User** somewhere safe, we'll use these soon!
```

Comment:
It would be great to encourage users to name their app password. See thread https://materializeinc.slack.com/archives/C06GZ7GBKB5/p1709593485646289 on how users can do that now.

Reply:
Not sure we should ask users to copy these, or just state that this is where to find the connection details you'll need in Step 2. The password is the only thing they need to store somewhere safe for later use.

Reply:
More than happy to re-word this, but copying them seemed like the easiest thing?